packages/kmath/src/line.js

```diff
@@ -15,10 +15,7 @@ export function distanceToPoint(line: Line, point: Point): number {
     return kpoint.distanceToLine(point, line);
 }
 
-export function reflectPoint(
-    line: Line,
-    point: Point,
-): $ReadOnlyArray<number> /* TODO: convert to Point */ {
```

Comment:
Now that `vector` uses a generic type, the incoming `Point` controls the arrity of the returned vector. :)